udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -62,6 +62,11 @@
   --text-color: #e7e7e7;
 }
 
+#quote-container {
+  margin-top: 20px;
+  color: #fff;
```

Comment:
Style prescribes alphabetical ordering of declarations.

Reply:
I had them alphabetical but separated by symbol, should they be alphabetical regardless of ./#/: ?